src/main/java/com/solvd/rating/web/dto/ExceptionDto.java

```diff
@@ -0,0 +1,12 @@
+package com.solvd.rating.web.dto;
+
+import lombok.Builder;
+
+@Builder
+public record ExceptionDto(
```

Comment:
can be named as `InternalError`. consider more simple naming for your classes. it can boost your maintainability